### ISSUE TITLE
Combine TimeZoneDsc Resource with ComputerManagementDsc 

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/xTimeZone#33]
As we are going to be changing this resource over to TimeZoneDsc in the near future it might make sense to move it over to make it part of ComputerManagementDsc during the move. I would also suggest doing the same with SystemLocaleDsc.

@kwirkykat - do you have any thoughts on this? I'm happy to keep them separate but I thought it might simplify things as these "individual" resources make more sense together.

And if we're implementing the fairly destructive changes of renaming the modules to *Dsc, this would seem to be the best time to do it.

**Comment by johlju** [comment]
I agree that they would fit better in ComputerManagementDsc. than have only one resource in a resource module. If the resource would have been called WindowsTimeDsc, then it could have been extended with more functionality for handling Windows Time Service. But even so, it would be better to have that in ComputerManagementDsc.
So I agree to the suggestion to move both xTimeZone and SystemLocaleDsc as soon as we are ready to rename xComputerManagement to ComputerManagementDsc.

**Comment by johlju** [comment]
We should update this resource module with a deprecated message once PR https://github.com/PowerShell/ComputerManagementDsc/pull/158 is merged. See example here:
https://github.com/PowerShell/xSqlPs

**Comment by johlju** [comment]
I think we should also update Maintainers.md in DscResources that instead of a maintainer it says 'DEPRECATED - moved to ComputerManagementDsc' or similar. 🤔

**Comment by PlagueHO** [comment]
Good idea - I'll get a PR ready to submit for it. I'm going to work on xPendingReboot next.
